lib/eex/lib/eex/compiler.ex

```diff
@@ -13,23 +13,26 @@ defmodule EEx.Compiler do
   def compile(source, opts) when is_binary(source) and is_list(opts) do
     file = opts[:file] || "nofile"
     line = opts[:line] || 1
+    column = 1
```

Comment:
I didn't have a use case for it yet, but in the future we might want to do this: `column = opts[:column] || 1` to allow user to adjust things.